impl/pom.xml

```diff
@@ -113,6 +113,7 @@
                 <version>1.4.3</version>
                 <configuration>
                     <instructions>
+                        <Automatic-Module-Name>java.el.impl</Automatic-Module-Name>
```

Comment:
There's a discussion going on about these module names right now. I had been asked to remove those for the time being in jaspic, JACC.